pkg/apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -188,6 +188,9 @@ type PipelineSpec struct {
 	// +required
 	InputRefs []string `json:"inputRefs"`
 
+	// Labels lists labels applied to this pipeline
+	// +optional
+	Labels map[string]string `json:"labels"`
```

Comment:
I see this results in `labels: null`  @periklis  @alanconway  doesn't `json:"labels,omitempty"` make it go away during serialization if empty?

Reply:
Fixed